### PR TITLE
Update documentation regarding dbt cloud prefix schema name

### PR DESCRIPTION
**Pull request by sebastianvillarroel**
## Pre-release docs
Is this change related to an unreleased version of dbt?
- [x] Unsure: we'll let you know!

### CODE REVIEW

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md [hunk 1]
```diff
@@ -15,10 +15,11 @@ To enable runs on Pull Requests, navigate to the Job Settings page for the relev
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/61536c9-Screen_Shot_2019-02-08_at_9.46.29_PM.png" title=""/>
 
+
 ## Understanding CI in dbt Cloud
 
-When Pull Request builds are enabled, dbt Cloud will listen for webhooks from GitHub indicating that a new PR has been opened or updated with new commits. When one of these webhooks is received, dbt Cloud will enqueue a new run of the specified job. Crucially, this run will be configured to build into a special, temporary schema. The name of these schema will be unique for each PR, and is shown in the Run Details page for the given run.
+When Pull Request builds are enabled, dbt Cloud will listen for webhooks from GitHub indicating that a new PR has been opened or updated with new commits. When one of these webhooks is received, dbt Cloud will enqueue a new run of the specified job. Crucially, this run will be configured to build into a special, [temporary schema](configure using-custom-schemas) using the prefix dbt_cloud. The name of these schema will be unique for each PR, and is shown in the Run Details page for the given run.
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md]:
```suggestion
When Pull Request builds are enabled, dbt Cloud will listen for webhooks from GitHub indicating that a new PR has been opened or updated with new commits. When one of these webhooks is received, dbt Cloud will enqueue a new run of the specified job. Crucially, this run will be configured to build into a special, [temporary schema](building-models/using-custom-schemas) using the prefix `dbt_cloud`. The name of these temporary schemas will be unique for each PR, and is shown in the Run Details page for the given run.
```

**Review by drewbanin — COMMENTED** [review]
Thanks for the PR @sebastianvillarroel - this is great! A couple of comments here to address a broken link, but otherwise, this LGTM :D

**Comment by sebastianvillarroel** [comment]
thanks @drewbanin for the commit suggestion :+1:

**Review by drewbanin — APPROVED** [review]
Thanks! Merging :)

**Pull request merged**